<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addStationWindow</class>
 <widget class="QDialog" name="addStationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="mainArea" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255, 255, 255, 0.9);
border-radius: 30px;</string>
   </property>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>490</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgba(195, 195, 195, 0.5);
height: 40px;
width: 150px;
font-family: 'MiSans';</string>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
font-family: 'MiSans Demibold';
font-size: 25px;</string>
    </property>
    <property name="text">
     <string>添加站点</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:'MiSans'; font-size: 20px;
background-color: rgba(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>名称</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:'MiSans'; font-size: 20px;
background-color: rgba(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>经度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:'MiSans'; font-size: 20px;
background-color: rgba(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>纬度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:'MiSans'; font-size: 20px;
background-color: rgba(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>标签位置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:'MiSans'; font-size: 20px;
background-color: rgba(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>站点形态</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'MiSans Light';
border-radius: 10px;
padding-left: 10px;
</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tagPosition">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'MiSans Light';
border-radius:0px;
padding-left:10px;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="stationType">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'MiSans Light';
border-radius:0px;
padding-left:10px;</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="longitude">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'MiSans Light';
padding-left:10px;</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="latitude">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'MiSans Light';
padding-left:10px;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
